server/repository/src/db_diesel/email_queue_row.rs

```diff
@@ -0,0 +1,111 @@
+use super::email_queue_row::email_queue::dsl::*;
+use crate::{RepositoryError, StorageConnection, Upsert};
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_derive_enum::DbEnum;
+
+table! {
+    email_queue (id) {
+        id -> Text,
+        to_address -> Text,
+        subject -> Text,
+        html_body -> Text,
+        text_body -> Text,
+        status -> crate::db_diesel::email_queue_row::EmailQueueStatusMapping,
+        created_at -> Timestamp,
+        updated_at -> Timestamp,
+        sent_at -> Nullable<Timestamp>,
+        retries -> Integer,
+        error -> Nullable<Text>,
+    }
+}
+
+#[derive(DbEnum, Debug, Clone, PartialEq, Eq, Hash)]
+#[DbValueStyle = "SCREAMING_SNAKE_CASE"]
+pub enum EmailQueueStatus {
+    Queued,
+    Sent,
+    Errored,
```

Comment:
```suggestion
    Errored, // Errored will be re-tried
    Failed,  // Failed will not be re-tried
```

I had this clarification in the notify version of this table, might be useful to copy here?

Reply:
Add here 56690bedfddd39b76e00ca771067e77ae9d102a6